maker-text.html

```diff
@@ -86,9 +86,6 @@ <h5 class="font-weight-light">
           <p class="">
             David Zhu and I recognized the opportunity to build a mobile app called FoodHabit, which users could leverage to easily and quickly log their meals. David was the genius who made our mobile application recognize the nutritional contents of a meal, only based on a photo of the plate. I helped users augment their will-power with features to make mindful eating easier. We wanted to afford users the ability to follow their desired eating habits by empowering them accurate information and healthy incentives.
```

Comment:
Who's `David Zhu`? For your audience, this new character seems out of place. You have the option of adding more or less context, and I think less favors you better in this situation.

Reply:
*worked on a team